eth/protocols/eth/handlers.go

```diff
@@ -398,33 +327,6 @@ func handleTransactions(backend Backend, msg Decoder, peer *Peer) error {
 	return backend.Handle(peer, &txs)
 }
 
-func handlePooledTransactions(backend Backend, msg Decoder, peer *Peer) error {
```

Comment:
I believe we still need this in order to fill our mempool with transactions.

Reply:
We have the replica of this function called `handlePooledTransactions66`, all i remove is the function used in ETH65 protocol version